src/utils/tracking.js

```diff
@@ -0,0 +1,104 @@
+const TRACKER_IDs = ['UA-100571693-1', 'UA-100571693-2']
+const PRODUCTION_API_URL = 'https://api.santiment.net'
+
+const isBrowser = typeof window !== 'undefined'
+const hasDoNotTrack = () => {
+  const dnt =
+    navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack
+  return dnt !== '1' && dnt !== 'yes'
+}
+// GA strings need to have trailing whitespace trimmed,
+function trim (s) {
+  return s.replace(/^\s+|\s+$/g, '')
+}
+
+function loadScript () {
+  const importGTAG = document.createElement('script')
+  importGTAG.async = true
+  importGTAG.type = 'text/javascript'
+  importGTAG.src = '//www.googletagmanager.com/gtag/js?id=' + TRACKER_IDs[0]
+
+  const head = document.getElementsByTagName('head')[0]
+  head.appendChild(importGTAG)
+}
+
+export function initializeTracking (trackerIDs = TRACKER_IDs) {
+  if (isBrowser && process.env.BACKEND_URL === PRODUCTION_API_URL) {
```

Comment:
Maybe need to use 'process.env.NODE_ENV === 'production'?

Reply:
@GarageInc stage server is production also.